source/Calamari.Tests/Fixtures/Certificates/WindowsX509CertificateStoreFixture.cs

```diff
@@ -221,12 +227,22 @@ public void ImportExistingCertificateShouldNotOverwriteExistingPrivateKeyRights(
                 Convert.FromBase64String(sampleCertificate.Base64Bytes()), sampleCertificate.Password,
                 storeLocation, storeName, sampleCertificate.HasPrivateKey);
 
-            var privateKeySecurity = new WindowsX509CertificateStore().GetPrivateKeySecurity(sampleCertificate.Thumbprint,
-                storeLocation, storeName);
+            var privateKeySecurity = CryptoKeySecurityAccessRules.GetPrivateKeySecurity(sampleCertificate.Thumbprint,
```

Comment:
Now that the `CryptoKeySecurityAccessRule` method is extracted, we can actually consolidate some of the test assertions that check for certificate access rules (See Below)